### ISSUE TITLE
`buffers` on shydactyl move on focus

### ISSUE THREAD

**Issue by bovine3dom** [cmcaine/tridactyl#581]
They aren't truncated properly like the history completions, either.

**Comment by glacambre** [comment]
That's because of https://github.com/cmcaine/tridactyl/blob/b5a3ac02102916313155ffa1bc6afdc2a758ae6d/src/completions.ts#L80 . It'll probably be fixed when [scrolloff](https://github.com/cmcaine/tridactyl/issues/27#issuecomment-357533100) is implemented.

**Comment by glacambre** [comment]
I can't reproduce this bug anymore on Tridactyl master + Firefox 62.0a1.

**Comment by bovine3dom** [comment]
I noticed it yesterday, so I think it's still there.

**Comment by glacambre** [comment]
You're right, I can still reproduce this bug on 1.13.1-39-gb026462 + Firefox 60.0.2.

Maybe the title of my tabs weren't long enough.

**Comment by bovine3dom** [comment]
The order / flex thing was something Shydactyl was already doing - or did you do something else?

**Comment by glacambre** [comment]
I fiddled with the `display` value of completion options and maybe also their widths, I don't remember exactly. All I really know is that the elements used to scroll to the left when selecting a completion option and now they don't anymore.

**Comment by bovine3dom** [comment]
Seems to have been fixed at some point in the last decade :)
